controlpanel/frontend/jinja2/datasource-detail.html

```diff
@@ -159,7 +156,7 @@ <h2 class="govuk-heading-m">Data access group</h2>
   {% endif %}
 </section>
 
-{% if user_is_admin %}
+{% if request.user.has_perm('api.destroy_s3bucket', object) %}
```

Comment:
Maybe we should have a permission viewing for the logs.
I know it's the same result but semantically these are different operations.

Potentially we may have different permissions to view logs.

A part from this YAGNI the other benefit would be that if someone tweaks the permissions required to delete an S3 bucket we don't end up also accidentally changing the permissions to view logs.

Reply:
yeah, i was being lazy - we have a permission for viewing app logs, but not for viewing s3 bucket logs